# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: uriloader/exthandler/ExternalHelperAppParent.cpp
# Commit: 74f99033251c
# Full Hash: 74f99033251cd6c36eb6b0b5a4713eb6b5c793fa
# Author: Jonathan Kingston <jkt@mozilla.com>
# Date: 2019-02-13 09:48:26
# Regressor Bug: 1520868
# File Overlap Count: 1
# Description:
#   Bug 1520868 - Replacing AsyncOpen2 with AsyncOpen always r=valentin
#   
#   Replacing js and text occurences of asyncOpen2
#   Replacing open2 with open
#   
# ==============================================================================

diff -r 3378fd5f4f22 -r 74f99033251c uriloader/exthandler/ExternalHelperAppParent.cpp
--- a/uriloader/exthandler/ExternalHelperAppParent.cpp	Tue Feb 12 17:58:19 2019 +0100
+++ b/uriloader/exthandler/ExternalHelperAppParent.cpp	Tue Feb 12 16:08:25 2019 +0000
@@ -277,18 +277,7 @@
 }
 
 NS_IMETHODIMP
-ExternalHelperAppParent::Open2(nsIInputStream** aStream) {
-  return NS_ERROR_NOT_IMPLEMENTED;
-}
-
-NS_IMETHODIMP
-ExternalHelperAppParent::AsyncOpen(nsIStreamListener* aListener,
-                                   nsISupports* aContext) {
-  return NS_ERROR_NOT_IMPLEMENTED;
-}
-
-NS_IMETHODIMP
-ExternalHelperAppParent::AsyncOpen2(nsIStreamListener* aListener) {
+ExternalHelperAppParent::AsyncOpen(nsIStreamListener* aListener) {
   return NS_ERROR_NOT_IMPLEMENTED;
 }
 